public/video-ui/src/components/Video/Display.js

```diff
@@ -41,20 +43,27 @@ class VideoDisplay extends React.Component {
 
     if(this.state.editable) {
       return (
-          <div className="container">
-            <form className="form">
-              <VideoEdit video={this.props.video || {}} updateVideo={this.updateVideo}/>
-              <SaveButton onSaveClick={this.saveVideo}/>
-            </form>
+          <div className="video">
```

Comment:
There's quite alot of duplication here between the two different renders... could there be a seperate video sidebar render method that does the editable check... or separate component that accepts a editable prop